src/oas/operation.ts

```diff
@@ -5,25 +5,29 @@ import { Spec } from 'swagger-schema-official';
 
 import { HttpOperationTransformer } from '../types';
 
-const methods = ['get', 'post', 'put', 'delete', 'options', 'head', 'patch', 'trace'];
+const DEFAULT_METHODS = ['get', 'post', 'put', 'delete', 'options', 'head', 'patch', 'trace'];
 
 export function transformOasOperations(
   document: Spec | OpenAPIObject,
   transformer: HttpOperationTransformer<any>,
+  methods: string[] | null = DEFAULT_METHODS,
```

Comment:
Is there a place where `null` is passed for `methods` ? I don't see any 🤷‍♂️ 

Seems to me this can be defined without it: 
```suggestion
  methods: string[] = DEFAULT_METHODS,
```

Reply:
Going to be useful in graphite, not in this repo.